db/migrate/20160524050538_add_conference_id_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddConferenceIdToVersions < ActiveRecord::Migration
+  def change
+    add_column :versions, :conference_id, :integer
```

Comment:
What about the old verion records that have nil for confernece_id? You should create a rake task to assign the conference_id for the old records too, when possible.